edenia/antelope-tools
# Feat/add a page for bp profile

### Add a Block Producer Profile page

### What does this PR do?

- Resolve #1303

### Steps to test

1. Run the project locally
2. Go to Block Producers page /block-producers
3. Click on view BP Profile
4. Check the content of the Block Producer Profile
5. Check meta data  

### Screenshots
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/cbc10841-1933-4601-bb73-d552fa68d4e6)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e682ff79-06ec-4a4e-8aba-4fa8a19ad50c)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/7b6a4833-7f4a-46ab-8db6-131edf2c474d)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/3ef818fe-8cd4-4853-af7a-6127f11a7098)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e25dd269-28d1-4cbf-8e4e-7038113f5747)
